### DISCUSSION TITLE
Reset model position when model changes

### DISCUSSION THREAD

**Discussion by furkandindar** [google/model-viewer#3748] [Q&A] [answered]
Hello,

I want to reset model position when a new model loads. Currently, if you rotate a model and then load a new model, newly loaded model is also rotated. You can check; https://joyous-plain-offer.glitch.me/ or simply open [modelviewer editor](https://modelviewer.dev/editor/) load an astronaut model, rotate it around and load a horse model. Horse model is going to load with the astronaut model's rotation. Is there any way to reset this so newly loaded model renders properly?

**Comment by elalish** [comment, marked as answer]
Just call `mv.cameraOrbit = "auto auto auto"` in your "load" listener.

**Reply by furkandindar** [reply]
thanks for the answer! I was able to figure it out yesterday, but now I have another issue, I want to limit the maximum zoom-in functionality. When I try it on the [Model Editor](https://modelviewer.dev/editor/)'s min zoom section it sets `min-camera-orbit="auto auto 0.5446m" min-field-of-view="20.94deg"` and it works. But when the model loads into the scene I'm guessing `min-camera-orbit="auto auto 0.5446m"` radius (0.5446m) causes model to render further away from its original position. And if I delete the `min-camera-orbit="auto auto 0.5446m"` attribute,  `min-field-of-view="20.94deg"` doesn't work at all. So I'm guessing I can't just use min-field-of-view to limit the zoom-in. can you help me with that?

**Reply by elalish** [reply]
Yes, the trouble is the radius; I think it'll work fine if you adjust `0.5446m` to an equivalent e.g. `50%`. That way when you load a differently-sized model, it will have a consistent relative zoom.

**Reply by furkandindar** [reply]
unfortunately changing radius to "50%" messes up the zoom-in functionality again..

**Reply by elalish** [reply]
Sorry, I didn't mean exactly 50%, but the appropriate % value to match the zoom-in that you want.

**Reply by furkandindar** [reply]
Thanks for the help, I think I'm close to figuring it out. appreciate it!
